File: src/Entity/Utilisateur.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Utilisateur
 *
 * @ORM\Table(name="utilisateur", indexes={@ORM\Index(name="equipe", columns={"equipe"})})
 * @ORM\Entity
 */
class Utilisateur
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="email", type="string", length=100, nullable=false)
     */
    private $email;

    /**
     * @var string
     *
     * @ORM\Column(name="password", type="string", length=32, nullable=false)
     */
    private $password;

    /**
     * @var string|null
     *
     * @ORM\Column(name="role", type="string", length=30, nullable=true)
     */
    private $role;

    /**
     * @var string|null
     *
     * @ORM\Column(name="nom", type="string", length=30, nullable=true)
     */
    private $nom;

    /**
     * @var string|null
     *
     * @ORM\Column(name="prenom", type="string", length=20, nullable=true)
     */
    private $prenom;

    /**
     * @var int|null
     *
     * @ORM\Column(name="categorie", type="integer", nullable=true)
     */
    private $categorie;

    /**
     * @var string|null
     *
     * @ORM\Column(name="raison_sociale", type="string", length=20, nullable=true)
     */
    private $raisonSociale;

    /**
     * @var string|null
     *
     * @ORM\Column(name="matricule_fiscale", type="string", length=20, nullable=true)
     */
    private $matriculeFiscale;

    /**
     * @var bool|null
     *
     * @ORM\Column(name="verifie", type="boolean", nullable=true)
     */
    private $verifie;

    /**
     * @var int|null
     *
     * @ORM\Column(name="solde_point", type="integer", nullable=true)
     */
    private $soldePoint;

    /**
     * @var string|null
     *
     * @ORM\Column(name="position", type="string", length=20, nullable=true)
     */
    private $position;

    /**
     * @var string|null
     *
     * @ORM\Column(name="telephone", type="string", length=8, nullable=true)
     */
    private $telephone;

    /**
     * @var int
     *
     * @ORM\Column(name="position_equipe", type="integer", nullable=false)
     */
    private $positionEquipe;

    /**
     * @var \Equipe
     *
     * @ORM\ManyToOne(targetEntity="Equipe")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="equipe", referencedColumnName="id")
     * })
     */
    private $equipe;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getPassword(): ?string
    {
        return $this->password;
    }

    public function setPassword(string $password): self
    {
        $this->password = $password;

        return $this;
    }

    public function getRole(): ?string
    {
        return $this->role;
    }

    public function setRole(?string $role): self
    {
        $this->role = $role;

        return $this;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(?string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(?string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getCategorie(): ?int
    {
        return $this->categorie;
    }

    public function setCategorie(?int $categorie): self
    {
        $this->categorie = $categorie;

        return $this;
    }

    public function getRaisonSociale(): ?string
    {
        return $this->raisonSociale;
    }

    public function setRaisonSociale(?string $raisonSociale): self
    {
        $this->raisonSociale = $raisonSociale;

        return $this;
    }

    public function getMatriculeFiscale(): ?string
    {
        return $this->matriculeFiscale;
    }

    public function setMatriculeFiscale(?string $matriculeFiscale): self
    {
        $this->matriculeFiscale = $matriculeFiscale;

        return $this;
    }

    public function getVerifie(): ?bool
    {
        return $this->verifie;
    }

    public function setVerifie(?bool $verifie): self
    {
        $this->verifie = $verifie;

        return $this;
    }

    public function getSoldePoint(): ?int
    {
        return $this->soldePoint;
    }

    public function setSoldePoint(?int $soldePoint): self
    {
        $this->soldePoint = $soldePoint;

        return $this;
    }

    public function getPosition(): ?string
    {
        return $this->position;
    }

    public function setPosition(?string $position): self
    {
        $this->position = $position;

        return $this;
    }

    public function getTelephone(): ?string
    {
        return $this->telephone;
    }

    public function setTelephone(?string $telephone): self
    {
        $this->telephone = $telephone;

        return $this;
    }

    public function getPositionEquipe(): ?int
    {
        return $this->positionEquipe;
    }

    public function setPositionEquipe(int $positionEquipe): self
    {
        $this->positionEquipe = $positionEquipe;

        return $this;
    }

    public function getEquipe(): ?Equipe
    {
        return $this->equipe;
    }

    public function setEquipe(?Equipe $equipe): self
    {
        $this->equipe = $equipe;

        return $this;
    }


}

File: src/entities/Utilisateur.java
package entities;



/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author sahar
 */
public class Utilisateur {
    int id;
    String nom;
    String prenom;
    Categorie categorie;
    Equipe equipe;
    String email;
    String roles;
    String pwd;
    String raisonSociale;
    String matriculeFiscale;
    int soldePoint;
    String position;
    String tel;
    int positionEquipe;
    boolean isVerified;

   

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public Categorie getCategorie() {
        return categorie;
    }

    public void setCategorie(Categorie categorie) {
        this.categorie = categorie;
    }

    public Equipe getEquipe() {
        return equipe;
    }

    public void setEquipe(Equipe equipe) {
        this.equipe = equipe;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getRoles() {
        return roles;
    }

    public void setRoles(String roles) {
        this.roles = roles;
    }

    public String getPwd() {
        return pwd;
    }

    public void setPwd(String pwd) {
        this.pwd = pwd;
    }

    public String getRaisonSociale() {
        return raisonSociale;
    }

    public void setRaisonSociale(String raisonSociale) {
        this.raisonSociale = raisonSociale;
    }

    public String getMatriculeFiscale() {
        return matriculeFiscale;
    }

    public void setMatriculeFiscale(String matriculeFiscale) {
        this.matriculeFiscale = matriculeFiscale;
    }

    public int getSoldePoint() {
        return soldePoint;
    }

    public void setSoldePoint(int soldePoint) {
        this.soldePoint = soldePoint;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getTel() {
        return tel;
    }

    public void setTel(String tel) {
        this.tel = tel;
    }

    public int getPositionEquipe() {
        return positionEquipe;
    }

    public void setPositionEquipe(int positionEquipe) {
        this.positionEquipe = positionEquipe;
    }
     public boolean isIsVerified() {
        return isVerified;
    }

    public void setIsVerified(boolean isVerified) {
        this.isVerified = isVerified;
    }
    
    public Utilisateur(){
        
    }
    
    public Utilisateur(int id,String email,String nom,String prenom,String position,String tel){
        this.id=id;
        this.nom=nom;
        this.prenom=prenom;
       this.position=position;
       this.tel=tel;
       this.email=email;

    }
    //client
    public Utilisateur(int id,String email,String pwd,String nom,String prenom,String position,String tel){
        this.id=id;
        this.email=email;
        this.pwd=pwd;
        this.nom=nom;
        this.prenom=prenom;
       this.position=position;
       this.tel=tel;
     

    }
    //inscrit client
    public Utilisateur(String email,String pwd,String nom,String prenom,String position,String tel){
        
        this.email=email;
        this.pwd=pwd;
        this.nom=nom;
        this.prenom=prenom;
       this.position=position;
       this.tel=tel;
     

    }
    
    //prop
     public Utilisateur(int id,String email,String raisonSociale,String matriculeFiscale,String tel){
        this.id=id;
        this.email=email;
        this.raisonSociale=raisonSociale;
        this.matriculeFiscale=matriculeFiscale;
        this.tel=tel;
     

    }//gestion prop
      public Utilisateur(String email,String raisonSociale,String matriculeFiscale,String tel){
 
        this.email=email;
        this.raisonSociale=raisonSociale;
        this.matriculeFiscale=matriculeFiscale;
        this.tel=tel;
     

    }
      //inscrit prop
        public Utilisateur(String email,String pwd,String raisonSociale,String matriculeFiscale,String tel){
 
        this.email=email;
         this.pwd=pwd;
        this.raisonSociale=raisonSociale;
        this.matriculeFiscale=matriculeFiscale;
        this.tel=tel;
     

    }
        //gestion arbitre
         public Utilisateur(String email,String nom,String prenom,Categorie categorie,String tel,String raisonSociale){
        
        this.email=email;
       
        this.nom=nom;
        this.prenom=prenom;
       this.position=position;
       this.tel=tel;
     

    }
         
         
         public Utilisateur(String email,String pwd,String nom,String prenom,Categorie categorie,String tel,String raisonSociale){
        
        this.email=email;
         this.pwd=pwd;
        this.nom=nom;
        this.prenom=prenom;
       this.position=position;
       this.tel=tel;
     

    }
         
      
      
    
    
   
    
    
    
    
    
}
